### ISSUE TITLE
Requesting func to retrieve current parameters of a connection

### ISSUE THREAD

**Issue by SMerrony** [distributed/sers#6]
I've switched https://github.com/SMerrony/DasherG to use this package for serial I/O and it's working well - thanks.

It would be useful to be able to query the current state of an active serial connection. Eg. a GetMode() func.

**Comment by distributed** [comment]
Hey! Yes, such a function is missing and is undoubtedly useful.

Currently, I am on vacation. I will get back to this in the following days.

Your terminal emulator is a very intriguing project and I am happy that sers is working for you:) What made you switch packages for serial access?

**Comment by SMerrony** [comment]
Thanks, no rush.  

I wrote up some notes on developing DasherG in Go: https://gomerrony.blogspot.com/2018/08/terminal-emulation-fun-with-go-gui.html

The main driver for switching to your package was proper BREAK support.

**Comment by distributed** [comment]
Hey!

Sorry, it took a bit longer than a couple of days to back to this.

Your terminal project - together with the maintenance of your minicomputer - are very cool projects indeed.

I guess I got lucky then, I added break support because someone else asked for it :)

All that being said, I have added a `GetMode()` method. The changes are in branch `getmode`, see merge request #7. Does this look reasonable to you?
